src/webots/control/WbControlledWorld.hpp

```diff
@@ -42,7 +42,7 @@ class WbControlledWorld : public WbSimulationWorld {
   void step() override;
 
   QList<WbController *> controllers() const { return mControllers; }
-  QList<WbController *> externControllers() const { return mExternControllers; }
+  QList<WbController *> externControllers() const { return mDisconnectedExternControllers; }
```

Comment:
Shall we also rename this method?
```suggestion
  QList<WbController *> disconnectedExternControllers() const { return mDisconnectedExternControllers; }
```

Reply:
I think it is a good idea, it will be more explicit.